src/css/our-lessons.css

```diff
@@ -0,0 +1,109 @@
+.lessons-list {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.lesson-item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 40px 20px;
+  border: 1px solid var(--green);
+  border-radius: 16px;
+}
+
+.lesson-item-title {
+  font-size: 20px;
+  font-weight: 700;
+  line-height: 1.2;
+  margin-bottom: 16px;
+}
+
+.lesson-item-price {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 1.2;
+  margin-bottom: 40px;
+}
+
+.lesson-item-text {
+  font-size: 14px;
+  line-height: 1.7;
+  margin-bottom: 20px;
+  color: var(--black);
+  opacity: 0.5;
+  text-align: center;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  margin-top: auto;
+}
+
+.lesson-item-button .button {
+  width: 100%;
+}
+
+.middle-item {
+  background-color: var(--green);
+  color: var(--white);
+}
+
+.middle-item .lesson-item-text {
+  color: var(--white);
+}
```

Comment:
opacity should be 0.7 and also middle-item is not a good classname, better to name it "active-item" or something like this

Reply:
Done